README.md

```diff
@@ -414,4 +423,20 @@ copycat.userAgent('foo')
 ```
 
 **note** For simplicity, this is currently working off of a list of 500 pre-defined user agent strings. If this is too limiting
-for your needs and you need something more dynamic than this, please let us know, and feel free to contribute :)
\ No newline at end of file
+for your needs and you need something more dynamic than this, please let us know, and feel free to contribute :)
+
+### `times(input, range, fn)`
```

Comment:
Just moved this to the bottom for now, considering its the most lower-level of our APIs. Still need to document this one better given how lower-level it is.